ui/app/routes/vault/cluster/secrets/backend/configuration/index.js

```diff
@@ -138,25 +128,23 @@ export default class SecretsBackendConfigurationRoute extends Route {
 
   async fetchAzureConfig(id) {
     try {
-      let response = await this.store.queryRecord('azure/config', { backend: id });
-      let isConfigSet = false;
-      // Azure will return a 200 if the config has not been set.
-      // Thus, we check every possible field and if any of them return a value we can assume the config has been set.
-      for (const property of POSSIBLE_AZURE_CONFIG_PROPS) {
-        if (response[property]) {
-          isConfigSet = true;
+      let azureModel = await this.store.queryRecord('azure/config', { backend: id });
+      let issuer = null;
+      if (azureModel.isConfigured) {
+        if (this.version.isEnterprise) {
```

Comment:
Could this be combined?
```suggestion
      if (azureModel.isConfigured && this.version.isEnterprise) {
```